### ISSUE TITLE
AnkiDroid randomly restarts

### ISSUE THREAD

**Issue by david-allison** [ankidroid/Anki-Android#15237]
I know this is closed... Should I report things like this anyway, on the closed issue, if I believe it to be the same? 
Edit: Sorry... this it's **very** old! It is likely not the same.

Anyway I've had four random restarts while using Anki, **after switching to the new "backend"** a few days ago (then I was still using the latest release from Google Play store). Before that I used the old backend for some months without noticing any similar problems.
After upgrading to the latest alpha version **(2.17 alpha 15)** yesterday, Anki has restarted once, which was about 20-40 minutes ago. I don't think my phone had internet connection at that moment, and I don't know if it will have sent an error report now, after connecting again.
I have no sounds on my card so that is not the problem.

AnkiDroid Version = 2.17alpha15 (e5b25f3fef737140ae396d1b212a8d638551500f)
Backend Version = 0.1.34-anki23.12.1 (23.12.1 1a1d4d5419c6b57ef3baf99c9d2d9cf85d36ae0a)
Android Version = 9 (SDK 28)
ProductFlavor = full
Manufacturer = LGE
Model = LG-H930
Hardware = joan
Webview User Agent = Mozilla/5.0 (Linux; Android 9; LG-H930 Build/PKQ1.190414.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/120.0.6099.210 Mobile Safari/537.36
ACRA UUID = 5b935c08-681b-4ebb-87c5-7701905caf7a
Crash Reports Enabled = true

_Originally posted by @andreas1974 in https://github.com/ankidroid/Anki-Android/issues/4097#issuecomment-1893256928_

**Comment by david-allison** [comment]
* https://ankidroid.org/acra/app/1/bug/116686/report/ed3439ed-a19b-4406-8267-48c60e501b12
   * `answerCard` - BackendInvalidInputException: card was modified - shouldn't be a crash 
* https://ankidroid.org/acra/app/1/bug/104505/report/44366cab-a291-40a3-8f7c-41e79a5559c0
  *  `answerCard` - BackendInvalidInputException: card was modified

**2.16.5**
* ~~https://ankidroid.org/acra/app/1/bug/57706/report/bafc1ef2-13b3-4882-81bf-0089731a3353~~
  * SyncStatus - BackendNetworkException: Please check your internet connection.
* ~~https://ankidroid.org/acra/app/1/bug/116014/report/62f2bf3c-1292-4672-a0f9-4ed83e6ee072~~ - no comment
* https://ankidroid.org/acra/app/1/bug/115976/report/053820dc-43a4-4e0f-b501-d31c7930e346
  * `answerCard` 
* https://ankidroid.org/acra/app/1/bug/57706/report/2ca9d808-ddd7-4506-ae2d-1e616bf65563
  * BackendNetworkException 

@andreas1974 I can' see any relevant restarts, only `card was modified`.

Do you have more information?

**Comment by andreas1974** [comment]
> BackendInvalidInputException: card was modified - shouldn't be a crash
You are right, that was not a crash or restart, just a popup with an error message that I reported. The restart occurred a little while later. I was in the process of adding a new Anki card. The unsaved card was lost upon the crash/restart. To add this card I had selected a text on Google or Wiki and I then I long-clicked the selection, using the popup menu to add _Anki card_ or _Anki note_ or what the name may be in English (the popup says **Anki-kort** in Swedish).

I don't believe my previous Ankidroid restarts occurred while adding cards. I'll try to make a better note the next time this happens and add it here.
